charts/logging-pipeline-plumber/templates/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+# apiVersion: v1
```

Comment:
this can be deleted

Reply:
this is actually needed, the service account needs a namespace. I forgot to find a way to do this proply. do you have any idea? like can we get the name space user pick with helm install there?

Reply:
you can use `{{ .Release.namespace }}`